packages/components/pie-input/src/index.ts

```diff
@@ -218,12 +218,15 @@ export class PieInput extends FormControlMixin(RtlMixin(LitElement)) implements
                     placeholder=${ifDefined(placeholder)}
                     ?readonly=${readonly}
                     ?required=${required}
+                    aria-describedby=${ifDefined(assistiveText ? assistiveTextIdValue : undefined)}
```

Comment:
Do you mind writing on the description what exactly we should hear/see on screen reader announcements? I get "invalid input" but no assistive text is read

Reply:
Have updated the description which includes a screenshot too.